events/elections/README.md

```diff
@@ -51,28 +53,31 @@ eligibility for voting, eligibility for candidacy, maximal representation, etc.
+- Description: Use the following text, modify it for either 3 or 4 positions, depending on the amount of open seats: This election is to nominate the steering committee for the Kubernetes project. Order the candidates by preference, the top $NUMBER candidates will be selected. Please see the voter's guide for more information.  PLEASE NOTE: "No opinion" is also a voting option if you do not feel comfortable ranking every single candidate
 - Add the candidate list to the form
 - How many choices will win: This number needs to be set to the amount of open seats of a given election
 - More options, check the boxes for:
-  - Do not release results to all voters.
-  - Enable detailed ballot reporting.
-  - Allow voters to select “no opinion” for some choices.
-- Click create poll, this will send election@kubernetes.io an email with instructions.
-- It will send you a link to “Poll Control”, bookmark this generated page as this is where you will add voters and also resend ballots to people if their ballot gets lost or filtered.
-- This page is where the “Start Poll” and “Stop Poll” buttons are, start the poll.
+  - Do not release results to all voters
+  - Enable detailed ballot reporting
+  - Allow voters to select “no opinion” for some choices
+- Click create poll, this will send election@kubernetes.io an email with instructions
+- It will send you a link to “Poll Control”, bookmark this generated page as this is where you will add voters and also resend ballots to people if their ballot gets lost or filtered
+- This page is where the “Start Poll” and “Stop Poll” buttons are, start the poll
 - WARNING: This is the point of no return: 
-  - Paste in the registered voters and click add voters.
-    - It will mail the ballots to the participants.
-    - It does duplicate detection so multiple entries are fine.
-    - This might take a while and the web page will not update, this has taken up to 10m in the past as it's sending each ballot. Don't panic or refresh the page. 
-- Leave the poll open for the duration of voting.
-  - Remember to send a 24 hour reminder before closing the poll.
-  - Click "Stop poll" at the end of the election, wait a minute then refresh the page for results. 
-  - Select "Condorcet IRV" on the right hand side of the page to select the results method.
-  - Mail results of the election to the steering committee private list and await instructions. 
-- Push election results into the community repo under events/elections _after_ the Steering Commmittee has announced the results. 
+  - Paste in the registered voters and click add voters
+    - It will mail the ballots to the participants
+    - It does duplicate detection so multiple entries are fine
+    - This might take a while and the web page will not update, this has taken up to 10m in the past as it's sending each ballot. Don't panic or refresh the page
```

Comment:
It took a couple days for all of the ballots to go out. Maybe we should say to estimate 2-3 minutes per voter, if CIVS is used again.

Reply:
I left this in as boilerplate, I have an item to talk to the GitHub folks during the contributor summit on figuring out if there's a way we can do an election using github primitives instead of email.